sdcflows/utils/wrangler.py

```diff
@@ -531,6 +540,39 @@ def find_estimators(
                     _log_debug_estimation(logger, e, layout.root)
                     estimators.append(e)
 
+                medic_entities = {**base_entities, **{'part': 'mag', 'echo': Query.ANY}}
```

Comment:
Okay, now this is going to lump together multiple echos. I guess we can't count on everybody doing `'echo': 1` for their first echo? But either way, we can resolve this by checking for whether a `mag_img` is already part of an estimator.

Reply:
And actually, maybe it makes sense to invert this and check for `phase`. You're less likely to get false positives that way.

Reply:
I'm fine hardcoding echo-1

Reply:
MEDIC uses all of the echoes for the field map estimation, so should all of the echoes (with both mag + phase) go into the `FieldmapEstimation`?